src/get_webhook_data.py

```
@@ -158,11 +169,56 @@ async def webhook(r: InputRequest):
+            
+            # Check if file already exists in repo
+            try:
+                csv_file = repo.get_contents(csv_path)
+                # File exists, update it
+                repo.update_file(
+                    csv_path,
+                    f"Updated {csv_filename} for {current_date}",
+                    csv_content,
+                    csv_file.sha,
+                )
+            except Exception:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)